src/dispatch/static/dispatch/src/router/config.js

```diff
@@ -46,7 +46,7 @@ export const publicRoute = [
   },
   {
     path: "/implicit/callback",
-    name: "PKCEImplicityCallback",
+    name: "PKCEImplicitlyCallback",
```

Comment:
Looks like linters caught a typo here. I don't know what the effect of this might be but highlighting it so we don't forget.